proto/environment/service.proto

```diff
@@ -118,7 +118,7 @@ message ListProjectsRequest {
   OrderDirection order_direction = 4;
   string search_keyword = 5;
   google.protobuf.BoolValue disabled = 6;
-  string organization_id = 7;
+  repeated string organization_ids = 7;
```

Comment:
This is a breaking change. Is this API not used by any other services? If  it is, we need to add a new field instead of modify it.

Reply:
I have already checked with him.
He is aware of breaking change, but since there is no place on the console that uses this property, there is no need to add a new one.